modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -507,7 +516,7 @@ CV_EXPORTS_W Rect selectROI(const String& windowName, InputArray img, bool showC
 CV_EXPORTS_W Rect selectROI(InputArray img, bool showCrosshair = true, bool fromCenter = false);
 
 /** @brief Selects ROIs on the given image.
-Function creates a window and allows user to select a ROIs using mouse.
+The function creates a window and allows the user to select a ROIs using mouse.
```

Comment:
Would change to "...select multiple ROIs..."

Reply:
maybe "select a ROI" AFAIK selectROI allows to select a single ROI